kda/scripts/verification.py

```diff
@@ -294,12 +294,12 @@ def check_transition_fluxes(G, kda_probs):
         kij = G[i][j][0]["val"]
         kji = G[j][i][0]["val"]
         # get the state probabilities for nodes i and j
-        pi = kda_probs[i]
-        pj = kda_probs[j]
+        pi = prob_arr[i]
+        pj = prob_arr[j]
         # calculate the transition fluxes in both directions
         J_ij = pi * kij
         J_ji = pj * kji
-        if not np.isclose(J_ij, J_ji):
+        if not np.isclose(J_ij, J_ji, rtol=1e-7, atol=1e-10):
```

Comment:
I made these tolerances more strict than the defaults by 2 orders of magnitude. Should have been in the commit message.